cpp/include/graph.hpp

```diff
@@ -416,6 +416,14 @@ class GraphCOO {
         rmm::device_buffer{graph.edge_data, graph.number_of_edges * sizeof(weight_t), stream, mr};
     }
   }
+  GraphCOO(GraphCOOContents<vertex_t, edge_t, weight_t> &&contents)
+    : number_of_vertices_p(contents.number_of_vertices),
+      number_of_edges_p(contents.number_of_edges),
+      src_indices_p(std::move(*contents.src_indices.release())),
```

Comment:
Maybe `*(contents.src_indices.release())` for readability to tell us the de-ref operator is on the pointer returned by the call to `release()`

Reply:
Added 0c6b9eba